### PR TITLE
initial commit - read or create & save tfidf wrapper object

### PR DESCRIPTION
**Pull request by user624086**

### CODE REVIEW

File: pygrams2.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def get_args(command_line_arguments):
 
 def main(supplied_args):
     paths = [os.path.join('outputs', 'reports'), os.path.join('outputs', 'wordclouds'),
-             os.path.join('outputs', 'table')]
+             os.path.join('outputs', 'table'), os.path.join('outputs', 'tfidf_wrapper')]
```
Comment (thanasions) [inline pygrams2.py]: can we change tfidf_wrapper to tfidf_obj

File: pygrams2.py [hunk 2]
```diff
@@ -94,7 +94,10 @@ def main(supplied_args):
     docs_mask_dict = argscheck.get_docs_mask_dict()
     terms_mask_dict = argscheck.get_terms_mask_dict()
 
+
     doc_source_file_name = os.path.join(args.path, args.doc_source)
+    tfidf_wrapper_filename = os.path.join('outputs', 'tfidf_wrapper', 'tfidf_wrapper.pickle')
```
Comment (thanasions) [inline pygrams2.py]: same here tfidf_wrapper to tfidf_obj

File: pygrams2.py (hunk 1 above)
Comment (user624086) [reply pygrams2.py]: Yes, I just wondered if people might confuse this object with the 'standard' sklearn tfidf object?

**Comment by user624086** [comment]
version 2 to come

**Pull request closed without merging**